.pre-commit-config.yaml

```diff
@@ -47,9 +47,8 @@ repos:
   - repo: https://github.com/golangci/golangci-lint
     rev: v1.59.1
     hooks:
-      - id: golangci-lint
+      - id: golangci-lint-full
```

Comment:
wow! didn't know about this - this is cool

Reply:
Yeah -- definitely a bit odd that we're just now finding this, but I assume most of the time we were probably testing the linter against committed/added files, so we wouldn't have otherwise known!

The main benefit here is that this actually works in CI. Previously to this change, you could run a `--no-verify` and push whatever you want up to the repo and CI wouldn't actually lint anything!